.github/workflows/nightly_clang_build.yml

```diff
@@ -8,7 +8,6 @@ on:
 jobs:
   build-using-clang:
     runs-on: ubuntu-latest
```

Comment:
@kenji-miyake  Thank you for the fixes.
I wanted to use `ubuntu:focal` because our development environment is Ubuntu 20.04 focal.
As described in the [official document](https://hub.docker.com/_/ubuntu), `ubuntu:latest` tag points to the latest LTS version, that is, it will be changed to another version a few years later. It might be no problem, but I would like to mention that here.

Reply:
@kmiya Thank you, I understand.
Though I wanted to use `ubuntu:focal`, but temporarily disabled it for consistency because some workflows didn't work with containers...
I'll try to solve the problem and revert `container` tag.